wordpress-mobile/WordPress-iOS
# Feature/4105 share settings

This patch implements sharing button management and other sharing settings. The feature is accessed from the sharing option on the blog details screen, then clicking on the manage button.

To test:
Edit the various none sharing button settings. Confirm they are updated in the blog.

Enable and disable the Twitter sharing button. Ensure the Twitter section is present when the button is enabled and absent when it is disabled.

Add buttons to the sharing and more sections. Ensure that buttons enabled in one are not enabled in the other. 

Add buttons to the sharing and more sections. Ensure they are enabled on the blog. 

Drag to reorder buttons in the sharing and more sections. Ensure they changed are reflected on the blog. 

Needs review: @alexcurylo
